freeipa/src/main/java/com/sequenceiq/freeipa/converter/instance/template/InstanceTemplateRequestToTemplateConverter.java

```diff
@@ -53,15 +53,19 @@
     private EntitlementService entitlementService;
```

Comment:
As I see we don't use `entitlementService` in this class. Can you please remove it?

Reply:
Also please remove the `LOGGER` since we don't use it here.